src/main/resources/launcher

```diff
@@ -40,4 +40,4 @@ if [ ! -f "${LAUNCHER_BIN}" ]; then
   exit 1
 fi
 
-"${LAUNCHER_BIN}" "$@"
+exxec "${LAUNCHER_BIN}" "$@"
```

Comment:
I assume this should be `exec`

Reply:
🤦 Yes, sorry. I tested this using a copy that I mounted in the Trino image.